<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1046</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="x_dockWidget">
   <property name="windowTitle">
    <string>X</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="x_dockWidgetContents">
    <property name="minimumSize">
     <size>
      <width>1026</width>
      <height>0</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="PlotWidget" name="x_PlotWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="r_dockWidget">
   <property name="windowTitle">
    <string>R</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="r_dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="PlotWidget" name="r_PlotWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="y_dockWidget_2">
   <property name="windowTitle">
    <string>Y</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="y_dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="PlotWidget" name="y_PlotWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="start_trace_Action"/>
   <addaction name="record_trace_Action"/>
  </widget>
  <action name="start_trace_Action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../artwork/icons/qudiTheme/22x22/start-counter.png</normaloff>
     <normalon>../artwork/icons/qudiTheme/22x22/stop-counter.png</normalon>../artwork/icons/qudiTheme/22x22/start-counter.png</iconset>
   </property>
   <property name="text">
    <string>Start trace</string>
   </property>
   <property name="toolTip">
    <string>Start trace</string>
   </property>
  </action>
  <action name="record_trace_Action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../artwork/icons/qudiTheme/22x22/record-counter.png</normaloff>
     <normalon>../artwork/icons/qudiTheme/22x22/stop-record-counter.png</normalon>../artwork/icons/qudiTheme/22x22/record-counter.png</iconset>
   </property>
   <property name="text">
    <string>Start recording</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
